src/common/components/clusterConfiguration/NetworkConfiguration.tsx

```diff
@@ -60,7 +60,7 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
   const isClusterCIDRIPv6 = Address6.isValid(values.clusterNetworkCidr || '');
   const { isIPv6, defaultNetworkType, isSDNSelectable } = React.useMemo(() => {
     const isIPv6 = isSubnetInIPv6({
-      machineNetworkCidr: values.machineNetworkCidr,
```

Comment:
we never store `machineNetworkCidr` in values. Only place to get it from is `cluster` object

Reply:
I may be wrong, but couldn't we set  `machineNetworkCidr` in  `getNetworkInitialValues` and then all the fields could be read from `values`? To avoid having this different behaviour on some fields.

Reply:
True, I'll keep that in mind, thanks!